pkg/config/config_test.go

```diff
@@ -28,22 +28,16 @@ import (
 )
 
```

Comment:
Can we have a test for both yaml and json?

Reply:
sure! on it